scripts/scil_tractogram_alter.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+"""
+This script is used to alter a bundle to reach specific minimum dice
+coefficient from the original bundle. The script will subsample, trim, cut,
+replace or transform the streamlines until the minimum dice is reached.
+(cannot be combined in one run, use the script multiple times if needed)
+
+All operation use a dichotomic search to find the parameter that gets as close
+as possible to the specified minimum dice coefficient (with an epsilon for
+convergence).
+
+The subsample operation will remove streamlines until the minimum dice is
+reached. This affect the whole bundle.
+
+The trim operation will used the lowest density voxels (starting at 1) to
+remove points from the streamlines until the minimum dice is reached.
+This typically affect the edge of the bundle.
+
+The cut operation will remove points from the start (or end) streamlines until
+the minimum dice is reached. This affect one end of the bundle.
```

Comment:
affects.

With the current description, I think that the difference between cut and trim is not clear. The vocabulary: edge of the bundle, end of the bundle, probably sound like the same thing to many users.